src/routes.rs

```diff
@@ -203,10 +203,34 @@ lazy_static! {
         index_path.push("index.html");
         index_path
     };
+    static ref JS_PATH: PathBuf = {
+        let mut js_path = PathBuf::new();
+        js_path.push(get_env!("CARGO_MANIFEST_DIR"));
+        js_path.push("resources");
+        js_path.push("highlight.min.js");
+        js_path
+    };
+    static ref ICON_PATH: PathBuf = {
+        let mut icon_path = PathBuf::new();
+        icon_path.push(get_env!("CARGO_MANIFEST_DIR"));
+        icon_path.push("resources");
+        icon_path.push("spectre-icons.min.css");
+        icon_path
+    };
+    static ref CSS_PATH: PathBuf = {
+        let mut css_path = PathBuf::new();
+        css_path.push(get_env!("CARGO_MANIFEST_DIR"));
+        css_path.push("resources");
+        css_path.push("spectre.min.css");
+        css_path
+    };
```

Comment:
Those should be extracted to a function, something like
```suggestion
    fn static_resource_path(filename: &str) -> PathBuf {
        let mut path = PathBuf::new();
        path.push(get_env!("CARGO_MANIFEST_DIR"));
        path.push("resources");
        path.push(filename);
        path
    }

    static ref JS_PATH: PathBuf = static_resource_path("highlight.min.js");
    static ref ICON_PATH: PathBuf = static_resource_path("spectre-icons.min.css");
    static ref CSS_PATH: PathBuf = static_resource_path("spectre.min.css");
```

Reply:
I wanted to implement your suggestions myself so I could get a little better at Rust. I have a question, since we're pulling this out to a function, wouldn't it also make sense to use this function for INDEX_PATH as well? If so, that's how I'll do it on my end.

Reply:
Yeah that'd make the most sense.